client/src/features/sessionsV2/SessionView/SessionView.tsx

```diff
@@ -190,6 +194,33 @@ function getSessionColor(state: string) {
     : "dark";
 }
 
+function SessionStartLink({
+  launcher,
+  project,
+}: Required<Pick<SessionViewProps, "launcher" | "project">>) {
+  const startPath = generatePath(
+    ABSOLUTE_ROUTES.v2.projects.show.sessions.start,
+    {
+      launcherId: launcher.id,
+      namespace: project.namespace,
+      slug: project.slug,
+    }
+  );
+  const { params } = useContext(AppContext);
+  const baseUrl = params?.BASE_URL ?? window.location.origin;
+  const url = `${baseUrl}${startPath}`;
```

Comment:
Use `new URL()` to construct a valid URL. `window.location.origin` will not contain a trailing slash:

 ```suggestion
  const url = new URL(startPath, params?.BASE_URL ?? window.location.href);
```